recipes/glibmm/all/conanfile.py

```diff
@@ -189,11 +184,11 @@ def package_info(self):
         if self._abi_version == "2.68":
             self.cpp_info.components[glibmm_component].requires = ["glib::gobject-2.0", "libsigcpp::sigc++"]
         else:
-            self.cpp_info.components[glibmm_component].requires = ["glib::gobject-2.0", "libsigcpp::sigc++-2.0"]
+            self.cpp_info.components[glibmm_component].requires = ["glib::gobject-2.0", "libsigcpp::libsigcpp"]
```

Comment:
If you look at libsigcpp recipe, this branching shouldn't exist, it should always be `self.cpp_info.components[glibmm_component].requires = ["glib::gobject-2.0", "libsigcpp::libsigcpp"]`. Existence of a component in libsigcpp is (or was) just a trick for target name of legacy cmake_find_package*.